frontend/src/metabase/nav/containers/MainNavbar/MainNavbarContainer/BookmarkList/BookmarkList.tsx

```diff
@@ -84,6 +84,8 @@ const BookmarkItem = ({
     bookmark.type === "collection" &&
     !PLUGIN_COLLECTIONS.isRegularCollection(bookmark);
 
+  const getIcon = () => (isSelected ? "bookmark_filled" : "bookmark");
```

Comment:
Same as the previous comment. Let's make this a variable. Also please note the different style of the functions. The previous one is using `return` while this one is using a shorthand return. Mostly we'd want things to be consistent. It's doesn't matter in this case if we would change it to a variable anyway, but that's something to keep in mind.